### DISCUSSION TITLE
Is the license `GPL-3.0-only` or  `GPL-3.0-or-later`

### DISCUSSION THREAD

**Discussion by yuioto** [xfangfang/wiliwili#355] [Q&A] [answered]
Arch Linux decided to use the SPDX license identifier in PKGBUILDs. However, SPDX license identifier requires that the license be `GPL-3.0-only` or `GPL-3.0-or-later`, but I did not find relevant instructions in the README.

More relevant information has been mentioned in this.
https://github.com/archlinuxcn/repo/issues/3587#issue-2083910430

**Comment by yuioto** [comment]
@xfangfang 可以使用GPL3更高的GPL许可证吗？
见过不同打包者选择了不同的许可证。

**Comment by xfangfang** [comment, marked as answer]
抱歉这两天事多错过这条通知了，对我来说这两个现在区别不大，但是因为之前打包 linux 安装包的时候设置了 GPL-3.0-only，避免改来改去，所以我认为答案应该是：GPL-3.0-only

https://github.com/xfangfang/wiliwili/blob/1662ec48ad73f3e18dff4f203b2e1c59d15c74b2/scripts/linux/cn.xfangfang.wiliwili.appdata.xml#L5

**Comment by Aleksanaa** [comment]
我是Nixpkgs的maintainer，感谢您

**Reply by xfangfang** [reply]
感谢维护，可以也把相关信息写在这里：https://github.com/xfangfang/wiliwili/discussions/92
